.github/workflows/build-test.yml

```diff
@@ -0,0 +1,49 @@
+name: Check
+
+on:
+  pull_request:
+
+  schedule:
+    - cron: "0 4 * * *" # Nightly build (@daily)
+
+concurrency:
+  # Only run once for latest commit per ref and cancel other (previous) runs.
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  check-code-style:
+    name: Code Style
+    uses: playframework/.github/.github/workflows/sbt.yml@v1
+    with:
+      cmd: sbt scalafmtCheckAll scalafmtSbtCheck
+
+  check-binary-compatibility:
+    name: Binary Compatibility
+    uses: playframework/.github/.github/workflows/binary-check.yml@v1
+
+  check-docs:
+    name: Docs
+    uses: playframework/.github/.github/workflows/sbt.yml@v1
+    with:
+      cmd: sbt docs/scalafmtCheckAll docs/scalafmtSbtCheck docs/test docs/validateDocs
+
+  tests:
+    name: Tests
+    needs: # Waiting more lightweight checks
+      - "check-code-style"
+      - "check-binary-compatibility"
+      - "check-docs"
+    uses: playframework/.github/.github/workflows/sbt-matrix.yml@v1
+    with:
+      java: >-
+        [ "11", "8" ]
+      scala: >-
+        [ "2.12.15", "2.13.8" ]
+      cmd: sbt ++$SCALA_VERSION test
```

Comment:
Funny, I can't see `test` even begin called in the old travis conf... 
Shouldn't `docs/test` be called here as well (in adition to `test`), instead above in `test-docs`? What do you think?

Reply:
`docs/test` feels more specific to the documentation, to me.  Even in the old travis config, it was called with a whole series of `docs` subproject commands.

I'm happy to change it, but I feel like it's a check specifically for documentation integrity, instead of runtime stability.

Reply:
You are absolutlely right. Its obvious to me now as well 😉